lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -30,7 +30,7 @@ def call(env)
         if !trace_id.sampled? || !routable_request?(env)
           @app.call(env)
         else
-          @tracer.with_new_span(trace_id, env[REQUEST_METHOD].to_s.downcase) do |span|
+          @tracer.with_new_span(trace_id, "#{env[REQUEST_METHOD].to_s.downcase} #{route(env)}".strip) do |span|
```

Comment:
maybe put in a unit test for this, as we don't want a span name of "get nil" or "get " If there's no route, the method alone should be the name.

Reply:
Done in latest commit, can you please review?